test/lib/SortedMerkleTree.test.ts

```diff
@@ -1,8 +1,23 @@
 import { expect } from 'chai';
 
 import { SortedMerkleTree } from '../../src';
+import { LeafValueCoder } from '../../src';
 
 describe('SortedMerkleTree', () => {
+  it('.squash root', () => {
```

Comment:
A minor nail polishing, but we normally use `#squashRoot` to describe the tested method on an instanced class. `.squashRoot` would be for a static method.